README.md

```diff
@@ -52,9 +52,11 @@ Inspired by the [Developer Conferences](https://github.com/MurtzaM/Developer-Con
 | [PennApps XIII](http://2016s.pennapps.com/) | Philadelphia, PA | 1.22 - 1.24 |
 | [UofTHacks](https://uofthacks.com/) | Toronto, Canada | 1.22 - 1.24 |
 | [SwampHacks](http://swamphacks.com/) | Gainesville, FL | 1.23 - 1.24 |
+| [Yale Healthcare Hackathon](http://www.yalehackhealth.org/) | New Haven, CT | 1.22-1.24 |
```

Comment:
Please move this above SwampHacks.

super nit: please add spacing for the date: "1.22 - 1.24". Thanks!